web/src/features/moreCast2/components/ForecastSummaryDataGrid.tsx

```diff
@@ -33,6 +39,19 @@ interface ForecastSummaryDataGridProps {
   handleClose: () => void
 }
 
+const getYesterdayRowID = (todayRow: MoreCast2Row): string => {
```

Comment:
All these helper functions could be pulled out into their own module with tests around them.